apps/desktop/src/renderer/index.css

```
@@ -2675,15 +2675,172 @@ button:active, [role="button"]:active {
+}
+
+.ade-work-tab--awaiting,
+.ade-work-lane-band--awaiting,
+.ade-work-lane-band-header--awaiting {
+  --awaiting-color: #F5A524;
   box-shadow:
-    inset 0 1px 0 color-mix(in srgb, white 10%, transparent),
-    0 0 0 1px color-mix(in srgb, var(--color-fg) 12%, transparent);
+    inset 0 0 0 1px color-mix(in srgb, var(--awaiting-color) 70%, transparent),
+    0 0 0 2px color-mix(in srgb, var(--awaiting-color) 35%, transparent),
+    0 0 14px color-mix(in srgb, var(--awaiting-color) 22%, transparent);
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)